partials/pagination.hbs

```diff
@@ -0,0 +1,20 @@
+<script type="text/javascript">
```

Comment:
Could this file be modified to use 4-space indentation?

Reply:
Fixed as part of be822a9f59639f12f436a55cedc415794b85a0d2